examples/torch/requirements.txt

```diff
@@ -7,3 +7,8 @@ mlflow>=1.12.1
 returns==0.14
 opencv-python>=4.4.0.46
 torchvision==0.10.1  # should always match the torch version that is installed via NNCF's setup.py
+efficientnet_pytorch
+
+# Please see
+# https://stackoverflow.com/questions/70520120/attributeerror-module-setuptools-distutils-has-no-attribute-version
+setuptools==59.5.0
```

Comment:
Could we bump the version of the setuptools in the setup.py of the NNCF to remove this dependence from here?

Reply:
Fixed.